contents/blog/posthog-vs-plausible.mdx

```diff
@@ -0,0 +1,178 @@
+  <ComparisonRow column1={true} column2={true} feature="Bounce rate" description="See the percentage of users that leave after one pageview" />
+  <ComparisonRow column1={true} column2={true} feature="Entry and exit paths" description="See the pages users first visit and the last ones before they leave" />
+  <ComparisonRow column1={true} column2={true} feature="Outbound clicks" description="See the links that take users away" />
+  <ComparisonRow column1={true} column2={true} feature="UTM tracking" description="Track marketing campaigns with UTM tags" />
+  <ComparisonRow column1={true} column2={true} feature="Cookieless tracking" description="Track users without cookies" />
+  <ComparisonRow column1={true} column2={true} feature="Snippet install" description="Install HTML snippet" />
+</ComparisonTable>
+
+## Product analytics
+
+For more customization, both in capturing data and analyzing it, teams rely on [product analytics](/product-analytics). Although not a focus of Plausible, it does have some product analytics features, but not nearly as many as PostHog's full suite. 
+
+Many of Plausible's product analytics features, like custom properties and [funnels](/docs/product-analytics/funnels), are on their more expensive "Business" plan. 
+
+<ComparisonTable column1="PostHog" column2="Plausible">
+  <ComparisonRow column1={true} column2={true} feature="Custom events" description="Capture and analyze any type of event" />
+  <ComparisonRow column1={true} column2="Events only" feature="Custom properties" description="Add more data to custom events or users" />
+  <ComparisonRow column1={true} column2={false} feature="Autocapture" description="Capture frontend clicks and events automatically" />
+  <ComparisonRow column1={true} column2={false} feature="Graphs and trends" description="Build custom insights and visualizations" />
+  <ComparisonRow column1={true} column2="Limited" feature="Funnels" description="Track users through a sequence of events" />
+  <ComparisonRow column1={true} column2={false} feature="Retention" description="Visualize which users stay, for how long" />
+  <ComparisonRow column1={true} column2={false} feature="Custom dashboards" description="Build a custom dashboard of your own or use a template" />
+  <ComparisonRow column1={true} column2={false} feature="Cohorts" description="Combine users based on properties and events for group analysis" />
+  <ComparisonRow column1={true} column2={false} feature="User profiles" description="View user demographic data such as country, language, and device type" />
+  <ComparisonRow column1={true} column2={false} feature="Group analytics" description="Track metrics at a company and account level" />
+  <ComparisonRow column1={true} column2={false} feature="Paths" description="Track user flows and where they drop off" />
+  <ComparisonRow column1={true} column2={false} feature="SQL" description="Query your data with SQL" />
+</ComparisonTable>
+
+## Platform
```

Comment:
We should add something around team member limits here. Plausible has them, we don't.